README.md

```diff
@@ -1,21 +1,36 @@
-# Welcome to The Odin Project Curriculum: [Learn Web Development for Free](http://theodinproject.com)
+# Welcome to The Odin Project Curriculum: 
 
-The Odin Project is an open source community and curriculum for learning what you need to know to be hireable as a web developer. Our curriculum consists of the best curated resources for learning web development on the web. We also have lots of original content throughout the courses and lessons. Each course has multiple projects that allow the students to practice what they have been learning and to use in their portfolio.
+The Odin Project (also known as TOP), is an open-source community for learning full-stack web development. Our mission is to be a continual provider for a free and comprehensive web development curriculum. We aim for students to become hireable web developers after having completed TOP's prescribed course of study.
```

Comment:
This doesn't read right to me. The continual bit seems off, but maybe thats just me.
"Our mission is to be a continual provider for a free and comprehensive web development curriculum"